beta/yarn.lock

```diff
@@ -467,6 +479,19 @@
     "@codemirror/view" "^0.19.0"
     "@lezer/javascript" "^0.15.1"
 
+"@codemirror/lang-markdown@^0.19.3":
```

Comment:
I don't know if it's useful, but from now on, Sandpack supports plain text and markdown files